nbconvert/exporters/slides.py

```diff
@@ -103,6 +103,21 @@ def _reveal_url_prefix_default(self):
         """
     ).tag(config=True)
 
+    reveal_transition = Unicode('linear',
+        help="""
+        Name of the reveal.js transition to use.
+
+        The list of themes that ship by default with reveal.js are:
+        default, cube, page, concave, zoom, none.
```

Comment:
Should this say 'list of *transitions*'? And shouldn't `linear` be in the list, as it's set as the default?